packages/apps/graph-client/src/pages/transaction/[key].tsx

```diff
@@ -0,0 +1,283 @@
+import { Notification, Table } from '@kadena/react-ui';
+
+import { useGetTransactionByRequestKeySubscription } from '../../__generated__/sdk';
+import Loader from '../../components/loader/loader';
+import { Text } from '../../components/text';
+import { styled } from '../../styles/stitches.config';
+
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import React from 'react';
+
+const StyledMain = styled('main', {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  flexDirection: 'column',
+  my: '5rem',
+});
+
```

Comment:
I can see that on the `src/components/loader/styles.css.ts` vanilla extract was used. Any reason to use stitches here?

Reply:
Good catch, copy pasted